nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -59,7 +59,13 @@
 
   <itemizedlist>
    <listitem>
-    <para />
+    <para>
+      <link linkend="opt-boot.zfs.forceImportAll">boot.zfs.forceImportAll</link>
+      previously did nothing, but has been fixed. However its default has been
+      changed to <literal>false</literal> to preserve the existing default
+      behaviour. If you have this explicitly set to <literal>true</literal>,
+      please note that your non-root pools will now be force imported.
```

Comment:
"forcibly imported" might sound a little bit better, but the documentation itself is clear enough and understandable. I had to read a little bit of ZFS documentation to understand why this might even be needed and what safeguards are bypassed by this option, but, well, we all learn something new every day, don't we?

Reply:
I agree with this. I'll merge once this is updated.

Reply:
Thanks, I've changed the wording. I understand the concern wrt. having to read ZFS documentation, but I'm a bit wary of including too much here, so I didn't make any other changes for now. :)